nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/io/marshallers/json/enrichers/FailingEnricher.java

```diff
@@ -43,6 +43,7 @@ public FailingEnricher() {
 
     @Override
     public void write(JsonGenerator jg, DocumentModel enriched) throws IOException {
+        jg.writeStringField("shouldnot", "bethere");
```

Comment:
To better simulate a failure with incomplete content I would rather do `jg.writeFieldName("justfieldname"); jg.flush()` so that we're sure the result is invalid.